README.md

```diff
@@ -2,24 +2,24 @@
 
-For watershed metrics that were not included in StreamCat (i.e., dominant Omernik Ecoregion, mean net primary production, and mean aridity index), we first delineated each sample's watershed using nhdplusTools, then utilized the terra package (Hijmans et al., 2022) to aggregate the additional datasets across each site's watershed accordingly. This approach is consistent with SteamCat's geospatial methodology.
+For watershed metrics that were not included in StreamCat (i.e., dominant Omernik Ecoregion, mean net primary production, and mean aridity index), we first identified each sample's watershed using {nhdplusTools}, then utilized the {terra} package (Hijmans et al., 2022) to aggregate the additional datasets across each site's watershed accordingly. This approach is consistent with SteamCat's geospatial methodology.
 
 Lastly, we collected streamflow data for sites that had a nearby stream gage. For locations without an identified co-located stream gage (WHONDRS typically co-located their sample sites with a stream gage), we identified USGS stream gages within 10 kilometers up- or downstream of our sampling locations using the dataRetrieval and nhdplusTools packages. All stream gages were then manually verified for their applicability to each sampling site (e.g., verifying there were no dams between the site and the stream gage, a major confluence, etc.). See Table 1 for a complete list of data sets included in this analysis.
 
-| Dataset                                          | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-|--------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Dataset | Description  |
```

Comment:
This is just me removing some white space. It will probably add it back in if you view it in the RStudio visual editor.